src/API/Api.js

```diff
@@ -0,0 +1,38 @@
+// const apiId = 'HCjJZpfpfCQz77Ezrye5';
```

Comment:
- [ ] For best practice do not commit old pieces of code as inline comments. [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=Do%20not%20commit%20old%20pieces%20of%20code%20as%20inline%20comments.%20They%20will%20make%20your%20project%20look%20messy.%20If%20you%20need%20to%20review%20a%20previous%20version%20of%20your%20code%2C%20you%20can%20always%20use%20git%20history.)

Reply:
@ahmetbozaci  thank you, old piece of code removed 
![e61](https://user-images.githubusercontent.com/40554384/167262523-3d00e1f5-8097-46e8-9330-7f1c36599684.PNG)